Api/Health Checks.md

```diff
@@ -0,0 +1,133 @@
+ASP.NET Core offers the built-in Health Checks feature for monitoring and reporting the health and availability of our application infrastructure components. Health checks provide a quick and automated way to verify that our application’s dependencies and components are running smoothly. They help us proactively identify issues within our application, allowing us to address them before they impact users. Regularly verifying the health of our application components can ensure a more reliable and resilient system.
+
+## Basic Health Check
+
+For many apps, a basic health probe configuration that reports the app's availability to process requests (liveness) is sufficient to discover the app's status.
```

Comment:
One small suggestion here - I would emphasize "availability to process requests" in some way (maybe by writing it in italic style) to make it clear that it is synonymous with the word in the brackets ("liveness").